### PR TITLE
Release 0.12.0

### LINKED ISSUE #230: Release files_pdfviewer 0.12.0
## Reason

After a series of important merges, a new release should be planned.
The last release from august has version 0.11.2
@micbar @jvillafanez 

- #228 - Update pdfjs to 2.5.
- #224 - make appstore fails on sign server
- #234 - Show Notification if downloading pdf is forbidden
- #237 - Enhance canDownload notification text
- #246 - Fix misalligned close X button
- #247 - Fix wrong locale settings
- #248 - Add translations

### Product approval 

- [ ] Verify transifex commits are in master: If a folder https://github.com/owncloud/<APPNAME>/tree/master/l10n exists, then https://drone.owncloud.com/owncloud/<APPNAME> should receive nightly '[tx] updated from transifex` ...
- [ ] Ping pmaier to check `info.xml` description and screenshot
- [ ] Get approval on translations from pmaier - send the date of the last tx commit from drone to Patrick

### QA

- [x] Generate Changelog
- [x] Create **QA** Milestone and set the description to $version
- [ ] Move items from `development` to `qa`
- [x] Create release branch with the format "release-$version"
- [x] Create "$version Testplan" ([QA templates](https://github.com/owncloud/QA/tree/master/Server)) #253 
- [x] copy new/reopened issues from the previous release ticket as 'Issues to review' below.

### Documentation

- [x] Changes in the docs needed? If so create issue in owncloud/docs.

### Marketing

- [x] Ping @owncloud/marketing

### Build

All actions to be done on the release branch from here:
- [x] Bump version in `info.xml` (no version suffix)
- [x] Check php version dependency in `info.xml`
- [x] Check minimum ownCloud version dependency in `info.xml`
- [x] Change CHANGELOG.md 'Unreleased' to '$version' (today's date)

#### Beta/RC

- [x] Create github tag for `v${version}RC1` on release branch
- [x] Build and sign RC tarball from github tag `v${version}RC1` (see
  handbook for how to build)
- [x] Upload as Github release using changelog entries
- [x] Test and document in testplan => Final "Go" from QA (set 'QA-ready' label) => else do another RC after fixing issues in the "release-$version" branch

#### Final

- [x] Check that no documentation issues are pending
- [x] Rename last RC tarball to final
- [x] Create github tag for final `v${version}` on release branch
- [x] Upload as Github release using changelog entries
- [x] Ping `#marketing` channel to coordinate publishing

### Publishing

- [x] Upload tarball to marketplace (see handbook in confluence for details)
- [x] Publish tarball for customers (see handbook in confluence for details) - https://customer.owncloud.com and https://portal.owncloud.com
- [x] Send mail to release-coordination with link to this ticket and to the changelog
- [x] Copy email to rocket chat `#updates` channel

### Post-release

- [x] Rename QA milestone to `$version`
- [x] Add a new 'Unreleased' section in CHANGELOG.md (and also update links at the bottom!)
- [x] Create PR to merge changes back to the `master` branch with
  description "Closes #XYZ" with the release ticket number
- [x] Close the milestone
- [x] Ensure [release template](https://github.com/owncloud/administration-internal/edit/master/tools/release/templates/server_app_release_template.md) is up to date

### PR DESCRIPTION
**Pull request by GeraldLeikam**
Merge back to Master
Closes #230

### CODE REVIEW

**Comment by JanAckermann** [comment]
Not pretty aware of how detailed the changelog Item should be, but we have translations now

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,9 +3,20 @@
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
-
 ## [Unreleased]
 
+## [0.12.0] - 2021-04-07
```
Comment (JanAckermann) [inline CHANGELOG.md:8]: Enhanche

**Review by JanAckermann — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (JanAckermann) [reply CHANGELOG.md:9]: ? I don't see the requested changes

Comment (GeraldLeikam) [reply CHANGELOG.md]: Fixed. thx

**Review by JanAckermann — APPROVED** [review]

**Pull request merged**